src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/samples/Sample1_HelloWorld.cs

```diff
@@ -16,25 +16,25 @@ public partial class ConfigurationSamples
         public async Task HelloWorld()
         {
             // Retrieve the connection string from the configuration store. 
-            // You can get the string from your Azure portal.
+            // You can get the string from your Azure portal or using Azure CLI.
             var connectionString = Environment.GetEnvironmentVariable("AZ_CONFIG_CONNECTION");
```

Comment:
Curious, since they're moving away from azconfig, what this environment variable could be?

Reply:
great point. What about `APP_CONFIG_CONNECTION`?. 
I will need to change it everywhere, so I'll do it in another PR